ccx_2.19.c

```diff
@@ -1445,8 +1446,41 @@ int main(int argc, char *argv[])
           printf("ERROR: This simulation type is not available with preCICE");
           exit(0);
         }
-      }
-
+      } else if (isModalDynamic){
+
+        printf("Starting FSI analysis via preCICE\n");
```

Comment:
Some print to acknowledge we use modal dynamic seems appropriate.

Reply:
Do you think this is really needed here? Thats something you configured in your CalculiX-.inp file and there is some output from CalculiX itself that it composes the dynamic response from the eigenmodes. We don't say that we use the dynamic modus either if we do so right? So on my opinion the information that belongs to preCICE is that we are doing an FSI analysis but i'm not sure if mentioning the method of the solid solver is appropriate here.

Reply:
Ah yes, if CalculiX says so, it's probably enough.